ansible/main.yml

```diff
@@ -1,8 +1,30 @@
 ---
-
 - name: Project Bootstrap
   hosts: all
   tasks:
+
+    - name: Bugzilla 1531511, 1476330, Github 18062 Workaround
+      block:
+        - name: Download Origin v3.6.1 client
+          get_url:
+            url: "https://github.com/openshift/origin/releases/download/v3.6.1/openshift-origin-client-tools-v3.6.1-008f2d5-linux-64bit.tar.gz"
+            dest: /tmp/oc-client.tar
+
+        - name: Extract oc
+          unarchive:
+            src: /tmp/oc-client.tar
+            dest: /tmp/
```

Comment:
Talking about site-specific breakage, isn't `/tmp` risky? noexec configurations, strange platforms, ... just wondering

Reply:
@codificat I am good to put it anywhere.  Somewhere else you think?